src/Conv.sol

```diff
@@ -0,0 +1,107 @@
+// SPDX-FileCopyrightText: 2025 Dai Foundation <www.daifoundation.org>
+// SPDX-License-Identifier: AGPL-3.0-or-later
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity ^0.8.24;
+
+/**
+ * @title Yearly Basis Points to per second RAY rate converter
+ * @notice Utility contract to convert between yearly basis points and per second RAY rate in both directions with full precision.
+ * @custom:authors [Oddaf]
+ * @custom:reviewers []
+ * @custom:auditors []
+ * @custom:bounties []
+ */
+contract Conv {
+    /// @notice The max bps supported in bps -> rate conversion.
+    uint256 public constant MAX = 50_00;
```

Comment:
```suggestion
    uint256 public constant MAX_BPS_IN = 50_00;
```

As pointed out in one of the audits, this limit only applies to `btor`, while `rtob` would work for any rate that doesn't cause an overflow in `rpow`.

We should probably rename this constant for clarity, since it's public viewable.